Automattic/jetpack
# Boost: Improve responsiveness during CCSS generation retry

Fixes #38169
Closes #40407

## Proposed changes:
* Optimistically update the UI to avoid any flicker when critical CSS generation is attempted after an error.

### Other information:


## Jetpack product discussion
None

## Does this pull request change what data or activity we track or use?
No

## Testing instructions:
- Use the Boost Dev plugin and set "Force Critical CSS generation Error" to timeout.
- Attempt C.CSS generation.
- It should fail, and you should see an error.
- Retry the generation.
- It should show the generating progress bar immediately without the UI glitch seen on #38169
- Make sure cloud CSS and local C.CSS are both working correctly in different situations after this change.
